src/main/java/com/hackclub/hccore/advancements/AstraAdv.java

```diff
@@ -0,0 +1,24 @@
+package com.hackclub.hccore.advancements;
+
+import com.fren_gor.ultimateAdvancementAPI.advancement.Advancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.BaseAdvancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplay;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplayBuilder;
+import com.fren_gor.ultimateAdvancementAPI.util.AdvancementKey;
+import com.fren_gor.ultimateAdvancementAPI.util.CoordAdapter;
+import com.hackclub.hccore.HCCorePlugin;
+import org.bukkit.Material;
+
+public class AstraAdv extends BaseAdvancement {
+    static AdvancementDisplayBuilder<AdvancementDisplay.Builder, AdvancementDisplay> displayBuilder = new AdvancementDisplay.Builder(Material.FIREWORK_ROCKET, "Ad " +
```

Comment:
can this be auto-granted based on y?
actually does this still use the previous update task? does that work?

Reply:
I'm wasn't sure what the conditions for this advancement to be granted were, so I left it as blank. What do you mean by the previous update task?

Reply:
https://github.com/hackclub/HCCore/blob/30d4d8e014634a280325e257a16c91f0a34ae51a/src/main/java/com/hackclub/hccore/tasks/CheckAdAstraTask.java